integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanRepaymentRescheduleAtDisbursementTest.java

```diff
@@ -134,11 +134,11 @@ public void testLoanRepaymentRescheduleAtDisbursement() {
         Calendar date = Calendar.getInstance(Utils.getTimeZoneOfTenant());
         date.set(2015, Calendar.MARCH, 16);
         expectedvalues.put("dueDate", getDateAsArray(date, 0));
-        expectedvalues.put("principalDue", "834.71");
```

Comment:
@josehernandezfintecheandomx Could you please tell more why this test was changed?

Reply:
It was set as original amount in the case